lib/src/util/class_names.dart

```diff
@@ -150,6 +166,32 @@ class ClassNameBuilder {
     _blacklistBuffer.write(className);
   }
 
+
+
+  /// Merges together the classes and blacklists present in another `ClassNameBuilder` instance.
```

Comment:
This should use a reference to `[other]`.

#nit "`ClassNameBuilder` instance" is redundant: See https://www.dartlang.org/guides/language/effective-dart/documentation#avoid-redundancy-with-the-surrounding-context

Also, "merges together" is a little vague. What about:
```dart
/// Merges the classes and blacklists from [other] into this builder.
```